src/form-snapshot.tsx

```diff
@@ -83,3 +95,21 @@ export const FormSnapshot: React.FC<Props> = ({
     </div>
   );
 };
+
+// Use this for more grammatically correct titles
+function getSingleProjectLabel(project: Project, msg: MsgFn) {
+  switch (project) {
+    case 'clothes_dryer':
+      return msg('clothes dryers');
```

Comment:
Best to put a description on these, so it's clearer to localizers what they're for:

`msg('clothes dryers', { desc: 'comes after "we found N savings programs for"' })`

and so on